src/pages/taskHistory/BuildVariantSelector.tsx

```diff
@@ -44,49 +42,43 @@ const BuildVariantSelector: React.FC<BuildVariantSelectorProps> = ({
       taskName,
     },
   });
+  const buildVariants = data?.buildVariantsForTaskName ?? [];
 
   const onChange = (selectedBuildVariants: string[]) => {
     updateQueryParams({
       [HistoryQueryParams.VisibleColumns]: selectedBuildVariants,
     });
   };
 
-  const { buildVariantsForTaskName } = data || {};
-
-  const handleSearch = (options: BuildVariantTuple[], match: string) =>
-    options.filter(
-      (option) =>
-        option.buildVariant.includes(match) ||
-        option.displayName.includes(match)
-    );
   return (
-    <Container>
-      <SearchableDropdown
-        data-cy="build-variant-selector"
-        label="Build Variant"
-        valuePlaceholder="Select Build Variant to View"
-        value={value}
-        onChange={onChange}
-        options={buildVariantsForTaskName}
-        disabled={loading}
-        allowMultiSelect
-        searchFunc={handleSearch}
-        optionRenderer={(option: BuildVariantTuple, onClick, isChecked) => (
-          <SearchableDropdownOption
-            key={`searchable_dropdown_option_${option.buildVariant}`}
-            value={option.buildVariant}
-            displayName={option.displayName}
-            onClick={() => onClick(option.buildVariant)}
-            isChecked={isChecked(option.buildVariant)}
-          />
-        )}
-      />
-    </Container>
+    <StyledCombobox
+      data-cy="build-variant-selector"
+      label="Build Variant"
+      placeholder="Select Build Variant to View"
+      disabled={loading}
+      overflow="scroll-x"
```

Comment:
`scroll-x` seems to break the padding for the placeholder.
![image](https://user-images.githubusercontent.com/4605522/160930554-06bdcb78-5cdd-4181-b1b2-3d708db1f392.png)
In the mean time how do you feel about manually padding the placeholder with spaces?
<img width="474" alt="image" src="https://user-images.githubusercontent.com/4605522/160930855-a9c50f7c-ccfa-4cbe-afbe-d9ab29e59e4d.png">